seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -502,16 +502,20 @@ angular.module('BE.seed.controller.mapping', [])
       };
 
       var get_geocoding_columns = function () {
-        organization_service.geocoding_columns(org_id).then(function (geocoding_columns) {
-          $scope.property_geocoding_columns_array = _.map(geocoding_columns.PropertyState, function (column_name) {
-            return _.find($scope.mappable_property_columns, {column_name: column_name}).display_name;
-          });
-          $scope.property_geocoding_columns = $scope.property_geocoding_columns_array.join(', ');
+        geocode_service.check_org_has_geocoding_enabled(org_id).then(function (result) {
+          if (result === true) {
+            organization_service.geocoding_columns(org_id).then(function (geocoding_columns) {
+              $scope.property_geocoding_columns_array = _.map(geocoding_columns.PropertyState, function (column_name) {
+                return _.find($scope.mappable_property_columns, {column_name: column_name}).display_name;
+              });
+              $scope.property_geocoding_columns = $scope.property_geocoding_columns_array.join(', ');
 
-          $scope.taxlot_geocoding_columns_array = _.map(geocoding_columns.TaxLotState, function (column_name) {
-            return _.find($scope.mappable_taxlot_columns, {column_name: column_name}).display_name;
-          });
-          $scope.taxlot_geocoding_columns = $scope.taxlot_geocoding_columns_array.join(', ');
+              $scope.taxlot_geocoding_columns_array = _.map(geocoding_columns.TaxLotState, function (column_name) {
+                return _.find($scope.mappable_taxlot_columns, {column_name: column_name}).display_name;
+              });
+              $scope.taxlot_geocoding_columns = $scope.taxlot_geocoding_columns_array.join(', ');
+            });
+          }
```

Comment:
Left is activated and right is deactivated. Nice catch to hide the relevant help text! 
<img width="1421" alt="Screen Shot 2020-11-11 at 5 29 35 PM" src="https://user-images.githubusercontent.com/30608004/98871947-bf8c8c00-2443-11eb-80bb-1f930b0cadd3.png">

Reply:
Added a message to the geocode modal if disabled.